lib/porter-util/index.d.ts

```diff
@@ -1,3 +1,4 @@
 /** biome-ignore-all lint/suspicious/noExplicitAny: laziness */
 export function writeArtifact(event: any, context: any): string;
 export function sendToDestination(event: any, body: any): void;
+export function binDir(bin: string): string;
```

Comment:
These are under a homebrew dir for me locally, so I made this configurable.

Reply:
ok, I added support for both a binDir, to give a common location, or to override individual binaries with their own env vars